linux_os/guide/services/ntp/chronyd_run_as_chrony_user/ansible/shared.yml

```diff
@@ -0,0 +1,30 @@
+# platform = multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+- name: "detect if file is not empty or missing"
+  find:
+    path: /etc/sysconfig/
+    patterns: chronyd
+    contains: '^([\s]*OPTIONS=["]?[^"]*)("?)'
+  register: chronyd_file
+
+- name: "replace existing setting or create a new file, rest is handled by different task"
```

Comment:
Task name example: `Update /etc/sysconfig/chronyd to run chronyd to as chrony user`.

Reply:
I think `create: True` is no necessary here, because the previous task found the file.